### ISSUE TITLE
Can the size of each mask be adjusted?

### ISSUE THREAD

**Issue by pouletdatrid** [keyu-tian/SparK#51]
I use the SIM super-resolution dataset to train Resnet50 because SIM images lack high-level semantic information and have low inter-block correlation, resulting in poor reconstruction quality. I think it may be necessary to have smaller mask blocks. For example, if the default size of each mask block in the code is (32, 32), is there a hyperparameter that can be adjusted to (16, 16)? Or do I need to modify the code myself, and if so, in which part of the code specifically?

**Comment by keyu-tian** [comment]
@pouletdatrid u can refer to our https://github.com/keyu-tian/SparK/tree/main/pretrain#some-details-how-we-mask-images-and-how-to-set-the-patch-size
